mod/metrics/lib/card/fixed_company_answer_query.rb

```diff
@@ -14,6 +14,7 @@ class FixedCompanyAnswerQuery < AnswerQuery
     # translate filter key to db column
 
     def initialize company_id, *args
+      @company_id = company_id
       @company_card = Card.fetch company_id
```

Comment:
I like the pattern of using the initialization mostly just to store the original info (in this case, the company_id) and using methods to populate other data as needed.  (eg, add `def company_card...`).  Just a thought. Not at all urgent, and definitely debatable.